composer.json

```diff
@@ -3,15 +3,19 @@
     "description": "GovCMS Drupal 8 Scaffold",
     "type": "project",
     "license": "GPL-2.0-or-later",
-    "repositories": {
-        "govcms": { "type": "composer", "url": "https://satis.govcms.gov.au/" },
-        "whitelist": { "type": "composer", "url": "https://satis.govcms.gov.au/whitelist/" },
-        "custom": { "type": "path", "url": "custom/composer" },
-        "packagist.org": false
-    },
+    "repositories": [
+        { "type": "path", "url": "custom/composer" },
+        { "type": "composer", "url": "https://packages.drupal.org/8" },
+        { "type": "composer", "url": "https://asset-packagist.org" },
+        { "type": "vcs", "url": "https://github.com/govcms/govcms8.git" },
+        { "type": "vcs", "url": "https://github.com/govcms/scaffold-tooling.git" },
+        { "type": "vcs", "url": "https://github.com/govcms/require-dev.git" }
```

Comment:
These are provided as custom repositories for now, rather than published to packagist.

A follow-up to reference packagist to follow (pending internal conversation & approvals).

Reply:
I don't think you need `{ "type": "vcs", "url": "https://github.com/govcms/govcms8.git" },` as it's already in a published package.